src/index.html

```diff
@@ -5,8 +5,387 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <div class="catalog">
+      <div class="menu catalog__menu">
+        <header class="header menu__header">
+          <a href="#home" class="header__logo">
+            <img src="./images/logo.png" alt="Mojo">
+          </a>
+          <nav class="header__nav nav">
+            <ul class="nav__ul">
+              <li>
+                <a
+                  class="nav__list is-active"
+                  href="#"
+                >
+                  Apple
+                </a>
+              </li>
+              <li>
+                <a
+                  class="nav__list"
+                  href="#"
+                >
+                  Samsung
+                </a>
+              </li>
+              <li>
+                <a
+                  class="nav__list"
+                  href="#"
+                >
+                  Smartphones
+                </a>
+              </li>
+              <li>
+                <a
+                  class="nav__list"
+                  data-qa="nav-hover"
+                  href="#"
+                >
+                  Laptops & Computers
+                </a>
+              </li>
+              <li>
+                <a
+                  class="nav__list"
+                  href="#"
+                >
+                  Gadgets
+                </a>
+              </li>
+              <li>
+                <a
+                  class="nav__list"
+                  href="#"
+                >
+                  Tablets
+                </a>
+              </li>
+              <li>
+                <a
+                  class="nav__list"
+                  href="#"
+                >
+                  Photo
+                </a>
+              </li>
+              <li>
+                <a
+                  class="nav__list"
+                  href="#"
+                >
+                  Video
+                </a>
+              </li>
+            </ul>
+          </nav>
+        </header>
+      </div>
+
+
+      <main class="main catalog__main">
+        <div class="first-four">
```

Comment:
don't create div for each row, you can make this rows using css instead of divs `first-four` and `second-four`